check_codegen.sh

```diff
@@ -1,12 +1,27 @@
 # This is a script which checks if yarn codegen was run it used by the check_codegen evergreen task
-BEFORE=$(git status --porcelain | grep src/gql/generated/types.ts)
-yarn codegen
-AFTER=$(git status --porcelain | grep src/gql/generated/types.ts)
+
+# path to generated types
+types_file="src/gql/generated/types.ts"
+BEFORE=$(git status --porcelain | grep "$types_file")
+yarn codegen || {
+  echo "script failed"
```

Comment:
nit. It might be nice if this was more explicit like `"yarn codegen failed to run"`, but it's up to you!